index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Portfolio Setup And Mobile Version Skeleton</title>
+  </head>
+  <body>
+    <section class="wrapper">
+      <header class="header">
+        <div class="logo"><img src="./images/my-logo.svg" alt="My Logo" /></div>
+        <nav class="icon">
+          <img src="./images/icon-menu.svg" alt="Icon Menu" />
+        </nav>
+      </header>
+      <div class="main">
+        <h1><br>I’m George<br>Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through <br>some of my work and experience! If you like what
+          you see and have a project you need<br> coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="connect">LET'S CONNECT</div>
+        <ul>
+          <li>
+            <a href="#"><img src="./images/angellist.svg" alt="Angellist" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/github.svg" alt="GitHub" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/linkedin.svg" alt="Linkedin" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/medium.svg" alt="Medium" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
+          </li>
+        </ul>
```

Comment:
- [ ] Please make sure that the social icons are properly aligned. Currently, some are not aligned.